api/controller/trigger.go

```diff
@@ -27,6 +29,9 @@ func saveTrigger(dataBase moira.Database, trigger *moira.Trigger, triggerID stri
 	if err := dataBase.AcquireTriggerCheckLock(triggerID, 10); err != nil {
 		return nil, api.ErrorInternalServer(err)
 	}
+	if err := checkExpressionValidity(trigger); err != nil {
```

Comment:
Same check is already done in https://github.com/moira-alert/moira/blob/a19b756b9ee9ef4f5ac3547e88e1dae688a2bb81/api/dto/triggers.go#L142

Why do you think it's going to help to make the same check twice?